test/deferred-open-test.js

```diff
@@ -169,5 +169,24 @@ buster.testCase('Deferred open()', {
         }
       })
     }
+  },
+
+  'queued operation is not serialized': function (done) {
+    var location = common.nextLocation()
+    var db = levelup(encDown(leveldown(location), { valueEncoding: 'json' }))
```

Comment:
You could do `{ keyEncoding: 'json' }` and also test the key

Reply:
Added a test.